README.md

```diff
@@ -12,3 +44,37 @@ Provide a detailed description and offer tools to implement product passports.
 
 We have an reference implementation which is the Digital Product Pass.
 You find the repository [here](https://github.com/eclipse-tractusx/digital-product-pass)
+
+### Documentation
+
+To get started you can have a look into our documentation:
+
+| Name                                                                      | Description                                                                                                                                                 |
+| ----------------------------------------------------------------          |-------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [Arc42](https://github.com/eclipse-tractusx/digital-product-pass/blob/main/docs/arc42/Arc42.md)                                             | Main Architecture Document (Arc42) of Digital Product Pass Application                                                                                      |
+| [Administration Guide](https://github.com/eclipse-tractusx/digital-product-pass/blob/main/docs/admin%20guide/Admin_Guide.md)                  | Administration Guide explaining the infrastructure and how to configure the application                                                                     |
+| [Data Retrieval Guide](https://github.com/eclipse-tractusx/digital-product-pass/blob/main/docs/data%20retrieval%20guide/DataRetrievalGuide.md)          | Backend documentation Product Passport App                                                                                                                  |
+| [Adoption View](/docs/page-adoption-view.md)                     | Technical Guide - Deployment in ArgoCD Hotel Budapest (integration environment)                                                                             |
+| [Development View](/docs/page-software-development-view.md)                                      | Overview on general docker commands                                                                                                                         |
+| [Operation View](/docs/page-software-operation-view.md)                     | This guide describes how to setup a keycloak instance in local docker container and import existing realm.json file.                                        |                                                                                                       |
+
+
+
+<!-- MARKDOWN LINKS & IMAGES -->
+<!-- https://www.markdownguide.org/basic-syntax/#reference-style-links -->
+
+[contributors-shield]: https://img.shields.io/github/contributors/eclipse-tractusx/digital-product-pass.svg?style=for-the-badge
```

Comment:
We are still pointing to digital product pass links, we need to add this badges from the eco pass kit